duckduckgo/iOS
# Remote Configuration

Task/Issue URL: https://app.asana.com/0/0/1200434558302342/f
Tech Design URL: https://app.asana.com/0/481882893211075/1200380127788005/f
CC: @brindy @bwaresiak @catehstn 

**Description**:
This PR integrates the new [remote configuration](https://github.com/duckduckgo/privacy-configuration) for privacy features and deprecates the legacy temp unprotected list.

**Steps to test this PR**:
1. Visit well known sites with trackers
2. Tracker count should match before and after changes
3. Visit https://privacy-test-pages.glitch.me/privacy-protections/fingerprinting/
4. Make sure `Include all headers and all properties of window, navigator and screen objects` is checked and run the test
5. Values overridden by `fingerprint.js` should match the results
6. Visit https://global-privacy-control.glitch.me/
7. Header and DOM signal should be present
8. Visit a [temp unprotected site](https://github.com/duckduckgo/privacy-configuration/blob/main/exception-lists/trackers-unprotected-temporary.json)
9. Site should display as "temporarily unprotected" in the privacy dashboard and should show trackers detected.



**Orientation Testing**:


**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
